src/cpp/dds/domain/DomainParticipant.cpp

```diff
@@ -32,7 +32,9 @@ namespace domain {
 DomainParticipant::DomainParticipant(
         uint32_t did)
     : dds::core::Reference<detail::DomainParticipant>(
-        eprosima::fastdds::dds::DomainParticipantFactory::get_instance()->create_participant(did))
+        eprosima::fastdds::dds::DomainParticipantFactory::get_instance()->create_participant(
+                did,
+                eprosima::fastdds::dds::DomainParticipantFactory::get_instance()->get_default_participant_qos()))
```

Comment:
Why not using the constant `PARTICIPANT_QOS_DEFAULT` here?

Reply:
Because I'm dumb.